h2o-algos/src/main/java/hex/tree/drf/DrfMojoWriter.java

```diff
@@ -15,7 +15,7 @@ public DrfMojoWriter() {}
   public DrfMojoWriter(DRFModel model) { super(model); }
 
   @Override public String mojoVersion() {
-    return "1.00";
+    return "0.10";
```

Comment:
@mmalohlava - I think the idea was that different algos could implement a different mojo API?